snap/snapcraft.yaml

```diff
@@ -56,12 +55,15 @@ parts:
     override-pull: |
       craftctl default
 
-      # set version and grade from code
+      # Checkout latest tagged version
       git config --global --add safe.directory '*'
+      latest_tag=$(git describe --tags)
+      echo "Checking out ${latest_tag}"
+      git checkout "${latest_tag}"
```

Comment:
If we stick to tags, we better make it explicit and use the `source-tag` entry of the part. This would also make it easier to automatise the release when there is a new tag upstream.

Reply:
I couldn't find a way with `source-tag` to always point to the latest one.

Reply:
There is none, point explicitly to the current latest one and we shall add later a job to alert us in case of a new upstream tag